[APP-3876] Set chat api model name to reserved DR name (#105)
datarobot-oss/qa-app-streamlit: README.md

A system prompt can be configured using the `SYSTEM_PROMPT` runtime parameter. This value will overwrite any previous prompts
set, including configuration from **Workbench > Playground**.

> The Q&amp;A app uses the reserved magic model name `datarobot-deployed-llm` for any Chat API calls. This is due to the 

Reviewer: ```suggestion
The Q&amp;A app uses the reserved model name `datarobot-deployed-llm` when making requests through the Chat API. This is due to the 
```

> openai client requiring the model parameter to be set. The Chat completion endpoint will then use the baked-in default model
> of the deployment. If you have multiple models within your deployment, you can modify this parameter by changing
> the `DEFAULT_CHAT_MODEL_NAME` in `constants.py`

Reviewer: ```suggestion
openai client requiring the model parameter to be set. By using this reserved value, the Chat completion endpoint will use the default model used when creating the custom model 
user in the deployment. If you have multiple models within your deployment, you can modify this parameter by changing
the `DEFAULT_CHAT_MODEL_NAME` in `constants.py`
```
